### ISSUE TITLE
The lowest version will be used if the version number is omitted

### ISSUE THREAD

**Issue by nowsprinting** [openupm/openupm-cli#156]
The lowest version will be used.

conditions:
- version number is omitted
- package name is start with `com.unity`

e.g.,
```
$ openupm add -f com.unity.test-framework
notice manifest modified com.unity.test-framework 1.4.2 => 0.0.4-preview
notice please open Unity project to apply changes
```

**Comment by favoyang** [comment]
I confirmed this is a bug introduced after `1.19.7`.

**Comment by favoyang** [comment]
Fixed in v1.19.11.

**Comment by nowsprinting** [comment]
@favoyang Thank you!
